lib/main.dart

```diff
@@ -28,9 +27,7 @@ class ScreenDecider extends StatelessWidget {
         primarySwatch: Colors.blue,
       ),
       debugShowCheckedModeBanner: false,
-      initialRoute: auth.getCurrentUser() == null
```

Comment:
@shazebK can you reset the initialRoute to its initial config please? Thanks! :+1: